scripts/a2_collections

```diff
@@ -73,3 +78,65 @@ a2 () {
   curl -sSkH "api-token: $TOK" -H "$projects" "$TARGET_HOST/$auth_path/$1" | jq $jq_option "$jq_script"
 }
 
+
+#############
+# authLoad
+#############
+# Purpose:
+#     Load a list of a2 auth resources.
+#     You can copy resources from one machine to another (e.g. for debugging or backup)
+#     by storing the output of authQuery into a file and feeding that file to authLoad.
+#
+# Options:
+#    -s or --show         show the executed curl commands
+#    -d or --dry-run      show the curl commands but do not execute them
+#    -2 or --v2           use IAM V2 endpoints
+#
+authLoad() {
+  # process options
+  dry_run=
+  show_cmd=
+  auth_path="api/v0/auth"
+  while [[ "$1" =~ ^- ]]; do
+    case "$1" in
+      -s|--show) show_cmd=1; shift;;
+      -d|--dry-run) dry_run=1; shift;;
+      -2|--v2) auth_path="apis/iam/v2beta"; shift;;
+      -*) echo "'$1' unknown"; return 1;;
+    esac
+  done
+
+  # prerequisites
+  ([ -z "$TOK" ] || [ -z "$TARGET_HOST" ]) && { echo "Need TOK and TARGET_HOST variables"; return; }
+  [ -z "$2" ] && { echo "usage: authLoad [ <options> ] <resource> <filename>"; return; }
+  if ! type jq &> /dev/null
+  then
+    echo "Need jq installed"; return;
+  fi
+  RESOURCE=$1
+  FILE=$2
+
+  # file-lines-to-array
+  old_IFS=$IFS
+  IFS=$'\n'
+  lines=($(cat $FILE))
+  IFS=$old_IFS
+
+  for line in "${lines[@]}"
+  do
+    if [ ! -z "$line" ]; then
```

Comment:
Since we are in bash, might we use `[[` here for consistency?

[info] `! -z` can be replaced by `-n` if you fancy.  I personally always have to look up which is which so I'm not sure that is actually a useful change though :D